WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -1028,13 +1029,23 @@ class MySiteViewModelTest : BaseUnitTest() {
     /* POST CARD - POST ITEM */
 
     @Test
-    fun `when post item is clicked, then post is opened for edit`() =
+    fun `given draft post card, when post item is clicked, then post is opened for edit draft`() =
             test {
                 initSelectedSite()
 
-                requireNotNull(onPostItemClick).invoke(postId)
+                requireNotNull(onPostItemClick).invoke(PostItemClickParams(PostCardType.DRAFT, postId))
 
-                assertThat(navigationActions).containsOnly(SiteNavigationAction.EditPost(site, postId))
+                assertThat(navigationActions).containsOnly(SiteNavigationAction.EditDraftPost(site, postId))
+            }
+
+    @Test
+    fun `given scheduled post card, when post item is clicked, then post is opened for edit scheduled`() =
+            test {
+                initSelectedSite()
+
+                requireNotNull(onPostItemClick).invoke(PostItemClickParams(PostCardType.SCHEDULED, postId))
+
+                assertThat(navigationActions).containsOnly(SiteNavigationAction.EditScheduledPost(site, postId))
```

Comment:
❓ I understand that `EditDraftPost` and `EditSchedulePost` are temporary navigation actions, but they do not open the editor yet. Do you think we should rename our tests to not include "post is opened" as it might be misleading in the present context?

Reply:
👋 @ashiagr !

Definitely, I was thinking the same, and I can rename it if you really think so, this is no problem. I personally didn't do that, because I wanted the `ViewModel` to be unaware of that and for it to push the expected action. The `Fragment` then, based on what it is available to it would decide which navigation to do. As such, I added a comment on the `Fragment` layer, specifying that the fragment actually ignores this intention, as it is not possible atm, and instead it navigates to the `Posts` screen. Wdyt?

Reply:
No worries, and that's why I left a question instead of a comment. We can live with it now that we've also discussed it in this PR. 

The PR is approved, feel free to merge it. 🎉

Reply:
🙇